eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -393,17 +409,47 @@ object Router {
     def cltvExpiryDelta: CltvExpiryDelta
   }
 
+  // @formatter:off
+  /** source of the channel routing parameters */
```

Comment:
The comment is a bit misleading now, maybe just:
```suggestion
  /** Channel routing parameters */
```

Reply:
Done with 6fe269081.